src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1179,6 +1179,9 @@ bool layout_optimizer::are_data_types_suitable_for_onednn(program_node& node) {
         if ((in_dt == data_types::i8 || in_dt == data_types::u8) && wei_dt == data_types::i8 &&
             (out_dt == data_types::f32 || out_dt == data_types::i32 || out_dt == data_types::f16 || out_dt == data_types::i8 || out_dt == data_types::u8))
             return true;
+        if ((in_dt == data_types::f32 && wei_dt == data_types::f32) &&
+            (out_dt == data_types::f32 || out_dt == data_types::i8 || out_dt == data_types::u8))
+            return true;
```

Comment:
This means that we will be using onednn for fp32 network. I'm afraid that this might have big performance impact as this is changing behavior a lot. Can we remove `out_dt == data_types::f32` so that f32->f32 can run in cldnn? Does it have negative impact on the target network?

Reply:
We can remove `out_dt == data_types::f32`. It does not impact on the target network.
![image](https://user-images.githubusercontent.com/20182877/170185433-7c24fdd8-6aa7-4c81-a3cf-334b35cead56.png)